conf/application.conf

```diff
@@ -171,14 +165,11 @@ Test {
   clam.antivirus {
     disableScanning = true
     runStub = false
-    chunkSize = 32768
 
     host = avscan
     port = 3310
 
     timeout = 5000
-    threadPoolSize = 20
-    maxLength = 25690112 // 24.5Mb
```

Comment:
were these removed for a reason?

Reply:
yes - they are not used by the latest version of clamav-client.